tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -357,3 +358,15 @@ Verify Image Can Be Spawned
     Spawn Notebook With Arguments    ${retries}    ${image}    ${size}
     ...    ${spawner_timeout}    ${gpus}    ${refresh}    &{envs}
     End Web Test
+
+Verify Jupyter Lab Version
+    [Arguments]  ${target}
+    ${ver} =  Run Cell And Get Output   !pip show JupyterLab | grep Version: | awk '{split($0,a); print a[2]}' | awk '{split($0,b,"."); printf "%s.%s.%s", b[1], b[2], b[3]}'
+    ${comparison} =  GTE  ${ver}  ${target}
+    [Return]  ${comparison}
+
+Verify Notebook Lab Version
+    [Arguments]  ${target}
+    ${ver} =  Run Cell And Get Output   !pip show notebook | grep Version: | awk '{split($0,a); print a[2]}' | awk '{split($0,b,"."); printf "%s.%s.%s", b[1], b[2], b[3]}'
+    ${comparison} =  GTE  ${ver}  ${target}
+    [Return]  ${comparison}
```

Comment:
ods-ci already contains keywords to check the version of lbraries. Pls have a look at `Verify Installed Library Version` and `Check Versions In JupyterLab`

Reply:
could you create a kw (e.g., `Verify Library Version Is Greater Than`) which takes as args the version and the lib name? so we at least avoid the code duplication between `Verify Jupyter Lab Version` and `Verify Notebook Lab Version`